includes/Core/Util/Entity_Factory.php

```diff
@@ -484,4 +491,73 @@ private static function prefix_title( $title, $prefix ) {
 			$title
 		);
 	}
+
+	/**
+	 * Converts given entity to amp entity if the given url is an amp url.
```

Comment:
Same here, upper case for acronyms:
```suggestion
	 * Converts given entity to AMP entity if the given URL is an AMP URL.
```